### PR TITLE
chore: upgrade to React Native 0.45

### PR DESCRIPTION
**Pull request by satya164**

### CODE REVIEW

File: src/components/Card/Card.js [hunk 1]
```diff
@@ -54,7 +54,7 @@ class Card extends Component<DefaultProps, Props, State> {
   constructor(props: Props) {
     super(props);
     this.state = {
-      elevation: new Animated.Value(props.elevation),
+      elevation: new Animated.Value(this.props.elevation),
```
Comment (ahmedlhanafy) [inline src/components/Card/Card.js]: @satya164 why did you change this?

Comment (satya164) [inline src/components/Card/Card.js]: No reason. Was trying to fix flow and this was left when committing.
